pbj-core/pbj-runtime/src/test/java/com/hedera/pbj/runtime/ProtoParserToolsTest.java

```diff
@@ -46,7 +67,7 @@ void testReadInt64() {
     @Test
     void testReadUint32() {
         testRead(() ->
-                rng.nextInt(0, Integer.MAX_VALUE),
+                rng.nextInt(0, MAX_VALUE),
```

Comment:
I like `Integer.MAX_VALUE` more than just `MAX_VALUE`. What if `Integer.MAX_VALUE ` and `Long.MAX_VALUE` are used in the same class? Leaving it up to you, though.

Reply:
Yes, qualified name makes sense in this case